app/assets/javascripts/tagging.js

```diff
@@ -2,6 +2,14 @@ function addTag(tagname, selector) {
 
   selector = selector || '#tagform';
 
+  if (tagname.indexOf("place") != -1) {
+    place = tagname.split(":")[1];
+    place.replace("-", " ");
+    geocodeStringAndPan(place);
```

Comment:
do we need to put the prompt code into a callback so it executes /after/ the geocode is complete? Or does this just work?

Reply:
Yes, perhaps in the `onComplete` parameter!

Reply:
I copied this code from blurred-location library, how do I test all of it?